Sources/BraintreeCore/BTGraphQLHTTP.m

```diff
@@ -63,11 +63,11 @@ - (void)handleRequestCompletion:(NSData *)data
 {
     // Network error
     if (error) {
-        [self callCompletionBlock:completionBlock body:nil response:nil error:error];
+        [self callCompletionBlock:completionBlock body:nil response:(NSHTTPURLResponse *)response error:error];
         return;
     }
 
-    NSDictionary *json = [NSJSONSerialization JSONObjectWithData:data options:0 error:NULL];
+    NSDictionary *json = [NSJSONSerialization JSONObjectWithData:data?:[NSData data] options:0 error:NULL];
```

Comment:
@jaxdesmarais To reproduce the crash (since OHHTTPStubs doesn't let you send nil data), you can add `data = nil` at the beginning of this function and then this line will crash with the message in the PR description.

Reply:
Was more curious about the payment flow you were going through to reproduce this crash but understand that this may have just been a general bug fix. We will get a PR up shortly for this.

Reply:
Oh sorry. Yeah I don't know, I'm just working off crash logs. It's somewhat rare, so it seems to be some sort of error state. I don't think it's something you could reliably reproduce (Maybe with a mock server or something that returns 500s)